src/views/components/dashboard/dashboard-item.vue

```diff
@@ -93,7 +93,7 @@ limitations under the License. -->
         };
       }
       if (!this.itemConfig.metricLabels && this.itemConfig.metricType === MetricsType.LABELED_VALUE) {
-        this.EDIT_COMP_CONFIG({ index: this.index, values: { metricLabels: 'p50, p75, p90, p95, p99' } });
+        this.EDIT_COMP_CONFIG({ index: this.index, values: { metricLabels: PercentileLabels } });
```

Comment:
This change is not right. The display should configure through UI by the end-users. Because label query covers more use cases than percentile.